nntrainer/src/neuralnet.cpp

```diff
@@ -172,6 +164,14 @@ int NeuralNetwork::setTrainConfig(std::vector<std::string> values) {
       continue_train = cont_train;
       opt.setProperty({values[i]});
     } break;
+    case PropertyType::batch_size: {
```

Comment:
How about neuralnet->setBatchSize() setting batch of layers and databuffer(if any) as well?

Reply:
Bigger suggestion:
How about s/`setTrainConfig`/`setRunConfig?Option?`  (or equivalent) and integrate inference option as well?

So that we can align with ml_train_model_run(...) api and add some run configs as well.
e.g.) 
"run mode=train/eval"
"metrics=accuracy"...?

Reply:
Updated to add neuralnetwork->setBatchSize()
For the bigger suggestion, opened #606.